doc/flat_fielding.rst

```diff
@@ -121,6 +121,21 @@ Of course, you will need to provide one or more images
 labeled as *illumflat* :doc:`frametype` in your :doc:`pypeit_file`.
 See below for further details.
 
+Lamps off Flats Subtraction
```

Comment:
Did this file not make it in the previous PR (MOSFIRE)?

Reply:
Yes, I realized, I only updated the mosfire doc, but lampoffflats can be used with any spectrograph.